internal/services/oracle/oracle_autonomous_database_clone_from_backup_data_source.go

```diff
@@ -0,0 +1,559 @@
+	Tags              map[string]string `tfschema:"tags"`
+
+	SourceAutonomousDatabaseId string `tfschema:"source_autonomous_database_id"`
+
+	// Base properties (computed)
+	AllowedIpAddresses                            []string                        `tfschema:"allowed_ip_addresses"`
+	BackupRetentionPeriodInDays                   int64                           `tfschema:"backup_retention_period_in_days"`
+	CharacterSet                                  string                          `tfschema:"character_set"`
+	ComputeCount                                  float64                         `tfschema:"compute_count"`
+	ComputeModel                                  string                          `tfschema:"compute_model"`
+	ConnectionStrings                             []string                        `tfschema:"connection_strings"`
+	CustomerContacts                              []string                        `tfschema:"customer_contacts"`
+	DataStorageSizeInGb                           int64                           `tfschema:"data_storage_size_in_gb"`
+	DataStorageSizeInTb                           int64                           `tfschema:"data_storage_size_in_tb"`
+	DatabaseVersion                               string                          `tfschema:"database_version"`
+	DatabaseWorkload                              string                          `tfschema:"database_workload"`
+	DisplayName                                   string                          `tfschema:"display_name"`
+	LicenseModel                                  string                          `tfschema:"license_model"`
+	AutoScalingEnabled                            bool                            `tfschema:"auto_scaling_enabled"`
+	AutoScalingForStorageEnabled                  bool                            `tfschema:"auto_scaling_for_storage_enabled"`
+	MtlsConnectionRequired                        bool                            `tfschema:"mtls_connection_required"`
+	NationalCharacterSet                          string                          `tfschema:"national_character_set"`
+	SubnetId                                      string                          `tfschema:"subnet_id"`
+	VnetId                                        string                          `tfschema:"virtual_network_id"`
+	LifecycleState                                string                          `tfschema:"lifecycle_state"`
+	PrivateEndpointUrl                            string                          `tfschema:"private_endpoint_url"`
+	PrivateEndpointIp                             string                          `tfschema:"private_endpoint_ip"`
+	ServiceConsoleUrl                             string                          `tfschema:"service_console_url"`
+	SqlWebDeveloperUrl                            string                          `tfschema:"sql_web_developer_url"`
+	TimeCreatedUtc                                string                          `tfschema:"time_created_utc"`
```

Comment:
could you also update other properties to make them consistent with the [naming guidance](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md)?
```suggestion
	TimeCreatedUtc                                string                          `tfschema:"time_created_in_utc"`
```